### PR TITLE
[FEED] - add employee and printed info.

### PR DESCRIPTION
**Pull request by abosiy**

### CODE REVIEW

File: src/com/bosiy/Employee.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    public Employee (String name, double salary, String position) {
+        this.name = name;
+        this.salary = salary;
+        this.position = position;
+    }
+
+    public String getName() {
+        return name;
+    }
```
Comment (scratchw0w) [inline src/com/bosiy/Employee.java:16]: invalid formatting

File: src/com/bosiy/Employee.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    public void setName(String name) {
+        this.name = name;
+    }
```
Comment (scratchw0w) [inline src/com/bosiy/Employee.java:20]: please add enter

**Review by scratchw0w — CHANGES_REQUESTED** [review]

File: src/com/bosiy/Employee.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package com.bosiy;
+
+public class Employee {
+
+    private String name;
+    private  double salary;
```
Comment (scratchw0w) [inline src/com/bosiy/Employee.java]: fix double spaces

File: src/com/bosiy/Employee.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    private  String position;
```
Comment (scratchw0w) [inline src/com/bosiy/Employee.java]: same

**Review by scratchw0w — CHANGES_REQUESTED** [review]

File: src/com/bosiy/Employee.java (hunk 1 above)
Comment (abosiy) [reply src/com/bosiy/Employee.java:16]: fixed

File: src/com/bosiy/Employee.java (hunk 2 above)
Comment (abosiy) [reply src/com/bosiy/Employee.java:20]: fixed

File: src/com/bosiy/Employee.java (hunk 3 above)
Comment (abosiy) [reply src/com/bosiy/Employee.java]: fixed

File: src/com/bosiy/Employee.java (hunk 4 above)
Comment (abosiy) [reply src/com/bosiy/Employee.java]: fixed

**Review by scratchw0w — APPROVED** [review]
Good Job!

**Pull request merged**